src/profile/helpers/__tests__/editData.test.ts

```diff
@@ -102,5 +165,187 @@ describe('editData.ts ', () => {
         });
       }
     });
+    it(`updateEditData clones and updates data when dataType is ${dataType} and returns formValues`, () => {
```

Comment:
Is the function being tested updateItemAndCreateSaveData? I wonder if updateEditData in this sentence should be replace with updateItemAndCreateSaveData?

Reply:
Fixed typo